scripts/calculate-advisors-rewards.py

```diff
@@ -0,0 +1,70 @@
+"""Usage: calculate-advisors-rewards.py --seed=STRING --proposals=FILE --advisors=FILE --total-incentive=INT [--output=FILE]
+
+Options:
+    --seed <string>  seed used for random selection of reviews
+    --proposals <file> file containing the reviews for each proposal
+    --advisors <file> file containing info for each advisor
+    --total-incentive <int> the amount of funding (in $) allocated to community advisors
+    --output <file> [ default: output.xlsx ]
+"""
+
+from docopt import docopt
+import random
+import pandas
+import math
+from collections import Counter
+
+NUM_WINNERS_PER_PROPOSAL = 3
+
+arguments = docopt(__doc__)
+proposals_file = pandas.read_excel(arguments["--proposals"], sheet_name=[0, 1])
+# 'ffil' fill empty columns using last valid value
+# (merged rows in spreadsheet are serialized with only the first row retaining the value)
+proposals = proposals_file[0].fillna(method="ffill")
+# Extract just email and wallet address (there does not seem to be a stable column name yet)
+advisors = {
+    k: v for k, v in pandas.read_excel(arguments["--advisors"]).iloc[:, 1:3].values
+}
+
+non_eligible_advisors = set(proposals_file[1].iloc[:, 0].values.tolist())
+
+seed = arguments["--seed"]
+random.seed(seed)
+print(f"Using seed: {seed}")
+
+winners = []
+for proposal_name, reviews in proposals.groupby(proposals.columns[0]):
+    # Advisor email in column F
+    assessors = set(pandas.unique(reviews.iloc[:, 5])) - non_eligible_advisors
+    proposal_winners = (
+        # Sort to ensure consistent set iteration
+        random.sample(list(sorted(assessors)), NUM_WINNERS_PER_PROPOSAL)
```

Comment:
`random` is documented to use the Mersenne Twister and the behavior of `random.sample` with a known seed can be assumed to be reproducible and platform-independent, correct?

Reply:
correct, at least in my understanding. Platform-independence is not explicitly mentioned in the docs but it does guarantee that the generator is Mersenne Twister, so I don't see why it should generate different values on different platforms.